<launch>
	<node name="jackal_mpc_track" pkg="mpc_trajectory_tracking" type="jackal_mpc_tracking" output="screen">
		<param name="mpc_steps" value="30" />
		<param name="x_goal" value="5"/>
		<param name="y_goal" value="3"/>
		<!-- <param name="theta_goal" value="-1.5707"/> -->
		<param name="theta_goal" value="0.0"/>
		<param name="w_linvel" value="10.0" />
		<param name="w_cte" value="4.0" />
		<param name="w_etheta" value="3.0" />

		<!-- <param name="w_angvel" value="1.0" />
		<param name="w_angvel_d" value="1.0" />
		<param name="w_linvel_d" value=".5" /> -->

		<param name="max_linvel" value="1.5" />
		<param name="max_angvel" value="1.0" />
		<param name="controller_frequency" value="10" />
	</node>
</launch>
